.github/workflows/cargo-update.yml

```
@@ -0,0 +1,27 @@
+---
+name: Test
```

```suggestion
name: update_deps
```
...or whatever...but there is already a `Test` workflow.